src/status_im/contexts/communities/actions/share_community_channel/style.cljs

```diff
@@ -30,6 +30,4 @@
 (defn gradient-cover-wrapper
   [width]
   {:width         (gradient-cover-size width)
-   :position      :absolute
-   :border-radius 12
-   :z-index       -1})
```

Comment:
ok moved from here. But still curious why added in first place

Reply:
yes, i also want to know the answer, because it just worked I didn't give too much attention to it, it might work wrong for sure, so this gradient component is tricky for sure, so it should be under every child of its parent